Compound.ipynb

```diff
@@ -468,6 +470,252 @@
     "plt.savefig(home+'/figure.png')"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Connect to IDR\n",
```

Comment:
```suggestion
    "### Create OMERO.figure\n",
```

The header is the most important part, it should reflect what is the content of this section of the notebook.